sites/footers/reports/a11y-1.json

```
@@ -0,0 +1,2032 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+
+  "@type": [ "earl:Assertion", "acr:AssessmentReport" ],
+
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://universallabs.org/wet/labs/pr2043/sites/footers/footers-en.html",
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "The redesigned 2022 site footer that do include an optional contextual footer",
+    "earl:pointer": {
+      "@type": "oa:CssSelector",
+      "@value": "footer#wb-info"
+    }
+  },
+
+  "earl:assertedBy": {
+    "foaf:name": "Service Canada - Principal Publisher",
+    "foaf:homepage": "https://github.com/ServiceCanada",
+    "@type": ["earl:Assertor", "foaf:Organization"],
+    "earl:mainAssertor": {
+      "foaf:name": "Brahim Mahadi Wachilli (Github: @BrahimMahadi)",
+      "foaf:homepage": "https://github.com/BrahimMahadi",
+      "@type": ["earl:Assertor", "foaf:Person"],
+      "foaf:member": {
+	    "@id": "_:accessibility_insights",
+        "doap:name": "Accessibility Insights for Web",
+        "doap:homepage": "https://accessibilityinsights.io/",
+        "doap:repository": "https://github.com/microsoft/accessibility-insights-web",
+        "doap:created": "2022-09-30",
+        "doap:release": "v2.35.0",
+        "@type": ["earl:Assertor", "earl:Software"]
+      }
+    }
+  },
+
+  "dct:date": "2022-12-02",
+
+  "dct:description": "Partial evaluation focused only on the component not its container. Evaluated all Success Criteria at Level AA of WCAG 2.2",
+
+  "acr:standard": "https://www.w3.org/TR/WCAG21",
+  "acr:conformanceOption": "act:standard/profiles/wcag#levelAA",
+
+  "earl:result": [
+    {
+      "@id": "_:assessment_tool_result",
+      "earl:test": "act_rules:wcag2x/automated/accessibility_insights_web",
+      "earl:subject": "_:subject",
+	  "earl:assertedBy": "_:accessibility_insights",
+      "earl:outcome": "earl:passed",
+      "dct:description": "",
+      "earl:mode": "earl:automatic",
+	  "dct:source": {
+		"@type": "acr:Asset",
```

```suggestion
		"@type": "acr:DataURL",
```

Il n'y a pas de classe "Asset" de definit dans le vocabulaire "acr", il y a seulement une propriété de même nom.

Je viens d'ajouter un data type nommé "DataURL" dans le vocabulaire "acr" qui est un alias vers la définition technique officielle de l'URL de donnée.

---

Addressed in commit #55fcb88